Documentation/getting_started.md

```diff
@@ -12,7 +12,7 @@ brew install openssl@1.1
 brew install mbedtls --HEAD
 brew install wget
 brew install qemu
-brew cask install docker
+brew install docker
```

Comment:
This doesn't look right. We should be using the `cask` version of docker. Recently brew has been updated to use `brew install --cask docker` instead.